src/components/common/Card.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+const CardWrapper  = styled.div`
+  padding: 10px;
+  margin-bottom: 5px;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  transition: 0.3s;
+`;
+
+const Card = ({ children }) => {
+	return (
+		<CardWrapper>
+			{children}
+		</CardWrapper>
+	)
+}
+
+export default Card;
```

Comment:
I think the card is too big, would be better to have more cards on the screen. 
![image](https://user-images.githubusercontent.com/26983421/164109676-32007848-cd64-44f5-b0d5-2850f8b5ab1b.png)